codeclimate_client/client.go

```diff
@@ -0,0 +1,42 @@
+package codeclimate_client
```

Comment:
go is a camcelCase language

Reply:
I'm not sure it will work, because it takes packages from GitHub and there is no camel case, so usually go packages are named without any cases and spaces, which look super weird, or using `-`, which is readable, but by some reason didn't work out for me, I'll try once again, otherwise I suggest to keep it as is.

Reply:
Actually, packages should be called with a single word:

https://blog.golang.org/package-names

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns.

Reply:
Discussed with Armen, he told that the standard is no camelcase, no underscore, no dash.